frame/alliance/src/tests.rs

```diff
@@ -441,9 +441,43 @@ fn retire_works() {
 			member: (3),
 			unreserved: None,
 		}));
+
+		// Move time on:
+		System::set_block_number(System::block_number() + RetirementPeriod::get());
+
+		assert_powerless(Origin::signed(3));
 	});
 }
 
+fn assert_powerless(user: Origin) {
+	//vote / veto with a valid propsal
+	let cid = test_cid();
+	let proposal = make_proposal(42);
+	
+	assert_noop!(Alliance::init_members(user.clone(), vec![], vec![], vec![]), BadOrigin);
+
+	assert_noop!(Alliance::set_rule(user.clone(), cid.clone()), BadOrigin);
+
+	assert_noop!(Alliance::retire(user.clone()), Error::<Test, ()>::RetirementNoticeNotGiven);
+
+	assert_noop!(Alliance::retirement_notice(user.clone()), Error::<Test, ()>::NotMember);
+
+	assert_noop!(Alliance::elevate_ally(user.clone(), 4), BadOrigin);
+
+	assert_noop!(Alliance::kick_member(user.clone(), 1), BadOrigin);
+
+	assert_noop!(Alliance::nominate_ally(user.clone(), 4),  Error::<Test, ()>::NoVotingRights);
+
+	assert_noop!(Alliance::propose(user.clone(), 5, Box::new(proposal), 1000),  Error::<Test, ()>::NoVotingRights);
+
+	// Currently failing below:
+	assert_noop!(Alliance::add_unscrupulous_items(user.clone(), vec![]), BadOrigin);
+
+	assert_noop!(Alliance::remove_unscrupulous_items(user.clone(), vec![]), BadOrigin);
+
+	assert_noop!(Alliance::announce(user, cid.clone()), Error::<Test, ()>::NotAlly);
```

Comment:
there three checked by `AnnouncementOrigin`, which is configurable by user, and for these tests its mocked to return true for user `3`